### PR TITLE
If user cancels login, show appropriate message and stop UI throbber

### PR DESCRIPTION
**Pull request by pspill**
Fixes NuGet/Home#1764

### CODE REVIEW

File: src/NuGet.Clients/PackageManagement.UI/Resources.resx [hunk 1]
```diff
@@ -511,4 +511,7 @@ Packages installed into Website projects will not be restored during build. Cons
     <value>not installed</value>
     <comment>In the solution package manager, if the select package is not installed, then this message is shown.</comment>
   </data>
+  <data name="Text_UserCancelled" xml:space="preserve">
+    <value>User cancelled operation</value>
```
Comment (yishaigalatzer) [inline src/NuGet.Clients/PackageManagement.UI/Resources.resx]: add a dot at the end of the message, And change to `operation canceled by user.`

File: src/NuGet.Clients/PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs [hunk 2]
```diff
@@ -165,8 +165,18 @@ private Task LoadAsync()
...
             {
+                if (!token.IsCancellationRequested)
+                {
+                    // The user cancelled the login, but treat as a load error
+                    // So the retry button and message is displayed
+                    _loadingStatusIndicator.Status = LoadingStatus.ErrorOccured;
+                    _loadingStatusIndicator.ErrorMessage = Resx.Resources.Text_UserCancelled;
+
+                    // Write stack to activity log
+                    Mvs.ActivityLog.LogError(LogEntrySource, ex.ToString());
```
Comment (yishaigalatzer) [inline src/NuGet.Clients/PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs]: if the user cancelled, why are we logging erros to the activity log?

Comment (pspill) [inline src/NuGet.Clients/PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs]: Agree, we don't need to log an error in this case.  Fixed.

**Comment by yishaigalatzer** [comment]
:shipit: 

@deepakaravindr can you take a look and merge this one as well?

**Comment by yishaigalatzer** [comment]
@deepakaravindr ping

**Comment by deepakaravindr** [comment]
Merged with https://github.com/NuGet/NuGet.Client/commit/c3a2abcedbcf3aa3014c08ac171747f28f69c81c

**Pull request closed without merging**